test/tests.js

```diff
@@ -29127,3 +29127,141 @@ test("/[a-z]/gim", {
 testFail("/[a-z]/u", "Invalid regular expression flag (1:1)");
 testFail("/[a-z]/y", "Invalid regular expression flag (1:1)");
 testFail("/[a-z]/s", "Invalid regular expression flag (1:1)");
+
+// https://github.com/ternjs/acorn/issues/255
+
+test("(a) = {}", {
```

Comment:
I added one for `(a.b) = {}` (https://github.com/ternjs/acorn/pull/433/files#diff-38dcab2adf8a7e3431829d8daca050fcR29190) but not the latter - can definitely add it